File: Programming Language(ver.2)/C++/408.cpp
#include <iostream>
using namespace std;

int main()
{
    string a, b;
    int i;
    cin >> a >> b;

    if (a.length() <= 3 || a.length() > 20) {
        cout << "error";
    }
    else if (b.length() <= 3 || b.length() > 20) {
        cout << "error";
    }
    else {
        cout << a.length() << endl;
        cout << b.length() << endl;
        a += b;
        for (i = a.length() - 1; i >= 0; i--) {
            cout << a[i];
        }
    }
    return (0);
}

File: Programming Language(ver.2)/PHP/408.php
<?php
    $str1 = trim(fgets(STDIN));
    $str2 = trim(fgets(STDIN));
    if(strlen($str1) < 3 || strlen($str1) > 20){
        printf("error\n");
        return;
    }
    if(strlen($str2) < 3 || strlen($str2) > 20){
        printf("error\n");
        return;
    }
    printf("%d\n", strlen($str1));
    printf("%d\n", strlen($str2));
    printf("%s\n", strrev($str1.$str2))
?>